apps/code-review/src/app/pages/overview/components/file-explorer/components/file-tree-node/file-tree-node.component.ts

```diff
@@ -92,6 +93,16 @@ export class FileTreeNodeComponent implements OnChanges, OnInit {
     public isOpened = false;
     public isSelected = false;
 
+    public readonly isMobile = this.deviceService.isMobile();
+
+    public readonly adaptiveRow = this.isMobile
```

Comment:
```suggestion
    private readonly rowAdaptationCssClass = this.isMobile
```

Зверни увагу на suggestion вище щодо перейменування поля і змінення модифікатору доступу.

Також,
Давай додамо виклик цього геттеру в іншому геттері `rowCssClassesAsString`:
```
const CLASSES = `monaco-tree-row ${this.rowAdaptationCssClass}`;
```